regression2.Rmd

```diff
@@ -129,7 +129,7 @@ $\beta_1$ as the increase in price for each square foot of space.
 Let's push this thought even further: what would happen in the equation for the line if you 
 tried to evaluate the price of a house with size 6 *million* square feet?
 Or what about *negative* 2,000 square feet? As it turns out, nothing in the formula breaks; linear
-regression will happily make predictions for crazy predictor values if you ask it to. But even though
+regression will happily make predictions for different predictor values if you ask it to. But even though
```

Comment:
"different" doesn't convey the right sentiment here -- instead of crazy, how about "nonsensical" ?

Reply:
Yeah "nonsensical" is a better word! Will change!